docs/conf.py

```diff
@@ -82,8 +82,8 @@ def __getattr__(cls, name):
 
 # General information about the project.
 project = u'geocat.comp'
-copyright = u'2019, University Corporation for Atmospheric Research'
-author = u'Orhan Eroglu'
+copyright = u'2021, University Corporation for Atmospheric Research'
```

Comment:
You could automate this with:

```suggestion
import datetime 
current_year = datetime.datetime.now().year
copyright = u'{}, University Corporation for Atmospheric Research'.format(current)
```